### ISSUE TITLE
lix download error when no haxe_libraries is present

### ISSUE THREAD

**Issue by markknol** [lix-pm/lix.client#45]
I got this `lix download` error when no haxe_libraries folder is present.

```
$ lix download
fs.js:904
  return binding.readdir(pathModule._makeLong(path), options.encoding);
                 ^

Error: ENOENT: no such file or directory, scandir 'C:\mark\projects\supersecretproj\haxe_libraries'
    at Object.fs.readdirSync (fs.js:904:18)
    at haxeshim_Scope.getInstallationInstructions (C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:4433:24)

    at haxeshim_HaxeCli.installLibs (C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:4096:22)
    at C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:5046:34
    at C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:8027:11
    at C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:7880:5
    at Object.DI3BfV/mmfQQnRVj5XUFw20N55EGIq3Pb67/UbzF5xE=.tink_core__$Callback_Callback_$Impl_$.invoke (C:\mark\project
s\supersecretproj\node_modules\lix\bin\lix.js:7481:3)
    at Object.DI3BfV/mmfQQnRVj5XUFw20N55EGIq3Pb67/UbzF5xE=.tink_core__$Callback_CallbackList_$Impl_$.invoke (C:\mark\pro
jects\supersecretproj\node_modules\lix\bin\lix.js:7579:42)
    at tink_core_FutureTrigger.trigger (C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:7895:46)
    at f (C:\mark\projects\supersecretproj\node_modules\lix\bin\lix.js:9478:242)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```
Using lix 15.1.0-rc.2

Would expect a more friendly warning

**Comment by nadako** [comment]
Is this released?

**Comment by kevinresol** [comment]
Just released 15.3.13

**Comment by markknol** [comment]
Thanks!
